chainermn/optimizers/__init__.py

```diff
@@ -0,0 +1,26 @@
+def create_multi_node_optimizer(actual_optimizer, communicator,
+                                double_buffering=False):
```

Comment:
We use a string argument to specify the communicator type in `create_communicator`. Here, should `double_buffering` be a boolean argument, or should we also use string argument here to specify the optimizer type? What do you think?

Reply:
If we plan to add another optimizer that can use double buffering, we should use the bool argument, because arguments are easy to understand. But, if the optimizer can not use double buffering, string argument may be acceptable. Do you have an idea of ​​new optimizer?

Reply:
Say, how about async SGD?

Reply:
Oh, It is important optimizer for us. Is not async and double buffering used at the same time?
If so, I think it would be better to string argument than boolean argument. What do you think?

Reply:
Discussed offline, and we concluded to use boolean argument.